diff --git a/tempest/api/network/test_floating_ips_negative.py b/tempest/api/network/test_floating_ips_negative.py
index 5ca17fe..2f75c78 100644
--- a/tempest/api/network/test_floating_ips_negative.py
+++ b/tempest/api/network/test_floating_ips_negative.py
@@ -54,6 +54,7 @@ class FloatingIPNegativeTestJSON(base.BaseNetworkTest):
         cls.create_router_interface(router['id'], subnet['id'])
         cls.port = cls.create_port(cls.network)

+    @decorators.skip_because(bug="1476935")
     @decorators.attr(type=['negative'])
     @decorators.idempotent_id('22996ea8-4a81-4b27-b6e1-fa5df92fa5e8')
     def test_create_floatingip_with_port_ext_net_unreachable(self):
@@ -73,6 +74,7 @@ class FloatingIPNegativeTestJSON(base.BaseNetworkTest):
                           fixed_ip_address=self.port['fixed_ips'][0]
                                                     ['ip_address'])

+    @decorators.skip_because(bug="1476935")
     @decorators.attr(type=['negative'])
     @decorators.idempotent_id('6b3b8797-6d43-4191-985c-c48b773eb429')
     def test_associate_floatingip_port_ext_net_unreachable(self):
