diff --git a/tempest/api/network/test_ports.py b/tempest/api/network/test_ports.py
index f81927d..33d6578 100644
--- a/tempest/api/network/test_ports.py
+++ b/tempest/api/network/test_ports.py
@@ -223,6 +223,7 @@ class PortsTestJSON(sec_base.BaseSecGroupTest):
         for port in ports:
             self.assertEqual(sorted(fields), sorted(port.keys()))

+    @decorators.skip_because(bug='1526260')
     @decorators.idempotent_id('63aeadd4-3b49-427f-a3b1-19ca81f06270')
     def test_create_update_port_with_second_ip(self):
         # Create a network with two subnets
