diff --git a/tempest/api/network/test_networks.py b/tempest/api/network/test_networks.py
index 3825f84ae6..782daa7230 100644
--- a/tempest/api/network/test_networks.py
+++ b/tempest/api/network/test_networks.py
@@ -23,6 +23,7 @@
 from tempest.lib.common.utils import test_utils
 from tempest.lib import exceptions as lib_exc
 from tempest import test
+from tempest.lib import decorators
 
 CONF = config.CONF
 
@@ -328,6 +329,7 @@ def test_create_delete_subnet_with_host_routes_and_dns_nameservers(self):
     def test_create_delete_subnet_with_dhcp_enabled(self):
         self._create_verify_delete_subnet(enable_dhcp=True)
 
+    @decorators.skip_because(bug="1617403")
     @test.idempotent_id('3d3852eb-3009-49ec-97ac-5ce83b73010a')
     def test_update_subnet_gw_dns_host_routes_dhcp(self):
         network = self.create_network()
diff --git a/tempest/api/network/test_routers_negative.py b/tempest/api/network/test_routers_negative.py
index cd9f6adadb..83edc89b21 100644
--- a/tempest/api/network/test_routers_negative.py
+++ b/tempest/api/network/test_routers_negative.py
@@ -20,6 +20,7 @@
 from tempest import config
 from tempest.lib import exceptions as lib_exc
 from tempest import test
+from tempest.lib import decorators
 
 CONF = config.CONF
 
@@ -132,6 +133,7 @@ def resource_setup(cls):
         cls.network = cls.create_network()
         cls.subnet = cls.create_subnet(cls.network)
 
+    @decorators.skip_because(bug="1617403")
     @test.attr(type=['negative'])
     @test.idempotent_id('4990b055-8fc7-48ab-bba7-aa28beaad0b9')
     def test_router_create_tenant_distributed_returns_forbidden(self):
